app/repositories/[id]/[...record]/page-content.tsx

```diff
@@ -103,17 +104,24 @@ export default function RecordViewPageContent({
           return undefined
         }
         // TODO: We need pagination here, right? how come getPostThread doesn't need it?
-        const { data: listData } = await client.api.app.bsky.graph.getList({
-          list: uri,
-        })
+        const { data: listData } = await client.api.app.bsky.graph.getList(
+          {
+            list: uri,
+          },
+          { headers: client.proxyHeaders() },
+        )
         return listData.items.map(({ subject }) => subject)
       }
-      const [record, profiles, thread] = await Promise.all([
+      const [record, profiles, thread] = await Promise.allSettled([
         getRecord(),
         getListProfiles(),
         getThread(),
       ])
-      return { record, thread, profiles }
+      return {
+        record: record.status === 'fulfilled' ? record.value : undefined,
+        thread: thread.status === 'fulfilled' ? thread.value : undefined,
+        profiles: profiles.status === 'fulfilled' ? profiles.value : undefined,
+      }
```

Comment:
By default I always avoid swallowing errors since it complicates debugging when an unexpected issue is going on.  Totally your call whether to handle that here (e.g. by logging the errors), but figured I'd mention it!

Reply:
we were kinda swallowing errors along this path already and the idea is that surfacing these errors were not helpful at all and would manifest as "failed to load record" or something along that line. most of these fetches are only adding additional content to the primary view so IMO this does not pose any issues.

Reply:
Just to clarify— I was imagining preserving the error for developers (not for users).  In my experience when an application has a habit of obscuring errors from devs it can become difficult to detect when an unanticipated issue is going on.

I recognize my style on this is maybe a little bit on the far end of the spectrum though 😆 With a few exceptional cases, I default to always ensure errors are either a. handled explicitly (e.g. ignoring specific anticipated error conditions) or b. logged.